google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/DebugTarget.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2015 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.gct.idea.debugger.ui;
+
+import com.google.api.services.clouddebugger.model.Debuggee;
+import com.google.common.base.Strings;
+import com.google.gct.idea.util.GctBundle;
+
+import com.intellij.openapi.diagnostic.Logger;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Map;
+
+/**
+ * The class models out the details for a target dubuggable module.
+ */
+class DebugTarget {
```

Comment:
It was refactored from being a private inner class of ProjectDebuggeeBinding which was a private inner class of CloudAttachDialog. So my thinking was to keep it as private as possible while still exposing it for testing.